projects/js-packages/publicize-components/src/components/review-prompt/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Panel that requests a review of the Jetpack Social Plugin
+ * Shows in the post publish panel of the editor
+ */
+
+import { Button, ThemeProvider } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { useCallback } from 'react';
+import useAnalyticsTracks from '../../hooks/use-anaytics-tracks';
+import styles from './styles.module.scss';
+
+const ReviewPrompt = props => {
+	const { href, onClose } = props;
+	const { recordEventHandler, recordEvent } = useAnalyticsTracks( {
+		pageViewEventName: 'social_plugin_review_prompt',
+		pageViewNamespace: 'jetpack',
+		pageViewSuffix: 'view',
+	} );
+	const recordReviewClick = recordEventHandler(
+		'jetpack_social_plugin_review_prompt_new_review_click',
+		{}
+	);
+
+	const handleDismiss = useCallback( () => {
+		recordEvent( 'jetpack_social_plugin_review_prompt_dismiss_click', {} );
+		onClose();
+	}, [ recordEvent, onClose ] );
+
+	return (
+		<ThemeProvider>
+			<div className={ styles.prompt }>
+				<p>
+					<strong className={ styles.header }>{ __( 'Presto!', 'jetpack' ) } 🎉</strong>
+				</p>
```

Comment:
The only reason I did not use a header initially was because I was unsure which level of header fit the existing page hierarchy. I agree though, using an `<hx>` tag would be preferable here.